.github/scripts/clean-build.sh

```diff
@@ -0,0 +1,229 @@
+# This script removes all files that shouldn't be included in a release
```

Comment:
```suggestion
#!/bin/bash
# This script removes all files that shouldn't be included in a release
```

I think we should add a shebang here to make sure it is always called with bash and not another shell.
Also am I missing something or is this file never called from the other scripts?